.github/scripts/run-docker.sh

```diff
@@ -7,17 +7,16 @@ CONFIG_VER=v0
 
 # Run Parameters
 RUN_SCRIPT=$1
-TODAY=$(date +"%Y%m%d")
 CONFIG_DIR=${PWD}/.github/configs/${CONFIG_VER}
 CONFIG_ENV=${CONFIG_DIR}/config-${CONFIG_VER}.env
-DATA_DIR=${HOME}/benchmark-results/${TODAY}_gh${GITHUB_RUN_ID}
```

Comment:
what is the contract around the 'gh' folder- it will be created fresh for each job, so TODAY is not needed?  or TODAY was more for archive, and not important now that you upload to github?

Reply:
Yes, it will be created for each job so TODAY is not needed.